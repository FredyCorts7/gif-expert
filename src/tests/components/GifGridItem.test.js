import React from 'react';
import { shallow } from 'enzyme';
import GifGridItem from '../../components/GifGridItem';
import '@testing-library/jest-dom';

describe('Testing to <GifGridItem/>', () => {
  test('should be the same component', () => {
    const title = 'Naruto';
    const url = 'naruto.jpg';
    const wrapper = shallow(<GifGridItem title={title} url={url} />);

    expect(wrapper).toMatchSnapshot();
  });
});
