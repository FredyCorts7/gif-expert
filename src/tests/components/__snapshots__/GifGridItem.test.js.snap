// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing to <GifGridItem/> should be the same component 1`] = `
<div
  className="card animate__animated animate__fadeIn"
>
  <img
    alt="Naruto"
    src="naruto.jpg"
  />
  <p>
    Naruto
  </p>
</div>
`;
